ambari-server/src/main/resources/Ambari-DDL-Derby-CREATE.sql

```diff
@@ -628,12 +628,12 @@ CREATE TABLE blueprint_service_config (
   CONSTRAINT FK_bp_svc_config_to_service FOREIGN KEY (service_id) REFERENCES blueprint_service (id));
 
 CREATE TABLE blueprint_mpack_configuration (
-  mpack_ref_id BIGINT NOT NULL,
+  mpack_instance_id BIGINT NOT NULL,
```

Comment:
Doesn't this require a change in `BlueprintMpackConfigEntity`, too?

https://github.com/apache/ambari/blob/80d840e5d50614a0f148a650c9bfc7d21363a218/ambari-server/src/main/java/org/apache/ambari/server/orm/entities/BlueprintMpackConfigEntity.java#L41

Reply:
Fixed. Thanks.